package.json

```diff
@@ -1,50 +1,21 @@
 {
-  "name": "edgar-search-client",
+  "name": "edgar-search-monrepo",
   "version": "1.0.0",
-  "description": "A search client for the SEC's Electronic Data Gathering, Analysis, and Retrieval (EDGAR) system.",
+  "description": "A monrepo containing a search API and a web client for the SEC's EDGAR filings site.",
   "main": "index.js",
-  "author": "Erik Eppel",
+  "repository": "https://github.com/epps/edgar-search-client.git",
+  "author": "Erik Eppel <eceppel@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "build:api": "yarn run clean:api && babel api --out-dir dist-api --extensions \".ts\"",
-    "watch:api": "babel api --out-dir dist-api --extensions \".ts\" --watch",
-    "clean:api": "rimraf dist-api",
-    "serve:api": "nodemon dist-api/index.js",
-    "lint:api": "eslint \"api/*.ts\" --quiet",
-    "lint:client": "eslint \"client/**/*.{js,jsx}\" --quiet"
-  },
-  "dependencies": {
-    "cheerio": "^1.0.0-rc.3",
-    "dotenv": "^8.0.0",
-    "express": "^4.17.1",
-    "node-fetch": "^2.6.0",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6"
+    "build:api": "yarn --cwd \"./packages/api/\" run build",
+    "serve:api": "yarn --cwd \"./packages/api/\" run serve",
+    "watch:api": "yarn --cwd \"./packages/api/\" run watch",
+    "clean:api": "yarn --cwd \"./packages/api/\" run clean",
+    "build:web": "yarn --cwd \"./packages/web-client\" run build",
+    "serve:web": "yarn --cwd \"./packages/web-client/\" run serve",
+    "install:all": "yarn --cwd \"./packages/api/\" install && yarn --cwd \"./packages/web-client/\" install"
```

Comment:
It appears this is the DIY version of handling package-specific commands. I stumbled upon this first and given it worked out-of-the-box without issues (and didn't seem to be forcing Yarn to do anything it wasn't meant to do) I went with it. However, in reading about deploying a monorepo Heroku, I stumbled upon an [article](https://medium.com/inato/how-to-setup-heroku-with-yarn-workspaces-d8eac0db0256) that pointed me to Yarn's [workspaces](https://yarnpkg.com/lang/en/docs/workspaces/). If time permits, I'd like to look into this.